### ISSUE TITLE
Request button jumps to the bottom after entering description when receiving money in a chat

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#18663]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open expensify app
2.Select a chat
3. Click on the + button
4. Click on request money
5. Enter amount and click next
6. Click on description
7. Click on the type bar and write something
8.  Click on Save ( and watch the receive money button jump to the bottom)
## Expected result:
The receive button should have been fixed at the bottom when clicking save. ( Works properly if we dont type anything in the description)
## Actual result:
Receive button jumps to the bottom half
## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android / native
- [x] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.12.0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/38ce324b-a6ac-43ce-bbd6-01f75a66e217

https://github.com/Expensify/App/assets/43996225/0dfc3f3b-7757-4006-a606-8540810799ee

**Expensify/Expensify Issue URL:**
**Issue reported by:** @SofoniasN
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1683203006095849

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by anmurali** [comment]
I am OOO today and tomorrow and then headed to Ec3, so I am unassigning myself

**Comment by slafortune** [comment]
Looks good!

**Comment by puneetlath** [comment]
I agree, doesn't seem like a bug to me. Comment or reopen if you disagree!
